pkg/config/config_parser.go

```diff
@@ -32,66 +39,59 @@ func (p *Parser) LoadConfigFile(path string) (*Config, diag.Diagnostics) {
 	return p.LoadConfigFromSource(contents)
 }
 
-func ValidateCQBlock(cq *CloudQuery) diag.Diagnostics {
+func HandleDecodedConfig(config *Config) diag.Diagnostics {
```

Comment:
Not a great name (`Handle` is too generic), maybe document?

Reply:
Good point. Improved it a bit in https://github.com/cloudquery/cloudquery/pull/913/commits/9850e6950466a4015d58386851e2cf233081cb68#diff-a985f21601efa99f89f2f497e2c36cfcb899000b335c36b09f830594ba49ad78R42.